packages/react-ui-kit/src/Form/Tooltip.tsx

```diff
@@ -21,64 +21,91 @@ import * as React from 'react';
 
 import {CSSObject} from '@emotion/react';
 
-import {COLOR} from '../Identity';
+import {Theme} from '../Layout';
 import {filterProps} from '../util';
 
 interface ToolTipProps<T = HTMLDivElement> extends React.HTMLProps<T> {
-  bottom?: boolean;
-  disabled?: boolean;
-  left?: boolean;
-  light?: boolean;
-  right?: boolean;
-  text?: string;
+  position?: 'top' | 'right' | 'left' | 'bottom';
+  body: React.ReactNode;
 }
 
-const tooltipStyle: <T>(props: ToolTipProps<T>) => CSSObject = ({
-  disabled = false,
-  bottom = false,
-  left = false,
-  right = false,
-  light = false,
-}) => ({
-  '&::after': {
-    backgroundColor: light ? COLOR.WHITE : COLOR.TEXT,
-    borderRadius: '4px',
-    bottom: bottom || left || right ? 'auto' : 'calc(100% + 8px)',
-    boxShadow: '0 2px 16px 0 rgba(0, 0, 0, 0.12)',
-    color: light ? COLOR.TEXT : COLOR.WHITE,
-    content: 'attr(data-text)',
-    display: 'block',
-    fontSize: '0.75rem',
-    fontWeight: light ? 400 : 600,
-    left: right ? 'calc(100% + 8px)' : 'auto',
-    lineHeight: '0.875rem',
-    maxWidth: '200px',
-    minWidth: '120px',
+const tooltipStyle: (theme: Theme) => CSSObject = theme => ({
+  position: 'relative',
+  width: 'fit-content',
+  '&:hover .tooltip-content': {visibility: 'visible', opacity: 1},
+  '.tooltip-content': {
+    textAlign: 'center',
+    visibility: 'hidden',
     opacity: 0,
-    padding: '12px',
-    pointerEvents: 'none',
+    width: 'max-content',
+    height: 'max-content',
     position: 'absolute',
-    right: left ? 'calc(100% + 8px)' : 'auto',
-    textAlign: 'center',
-    top: bottom ? 'calc(100% + 8px)' : 'auto',
-    transform: left || right ? `translateX(${left ? -16 : 16}px)` : `translateY(${bottom ? -16 : 16}px)`,
-    transition: 'all 0.15s ease-in-out',
+    boxSizing: 'border-box',
+    display: 'block',
+    margin: '0 auto',
+    padding: '4px 8px',
+    backgroundColor: theme.Tooltip.backgroundColor,
+    color: theme.Tooltip.color,
+    borderRadius: 4,
+    fontSize: '0.8em',
```

Comment:
can't this be something like `theme.fontSizes.small`  (0.75rem)? 
rem would let it scale with the font size slider, design was talking about using tooltips for accessibility, that would help

Reply:
sure, done in:
https://github.com/wireapp/wire-web-packages/pull/5012/commits/778bca21cb9eaea35dca9f16be11ff061b68fdad